{{cookiecutter.project_name}}/CONTRIBUTING.md

```diff
@@ -53,51 +51,52 @@ If you are proposing a feature:
 
 Ready to contribute? Here's how to set up **{{cookiecutter.project_name }}** for local development.
 
-1. Fork the **{{ cookiecutter.project_name }}** repo on {{ cookiecutter.vs | title }}.
-
-2. Clone your fork locally:
+1. Fork the **{{ cookiecutter.project_name }}** repository.
 
-        $ git clone ssh://git@{{ cookiecutter.vs_url }}{% if cookiecutter.vs|lower == "bitbucket" %}:7999/~{% else %}/{% endif %}your_name_here/{{ cookiecutter.project_name }}.git
+1. Clone your fork locally.
 
-3. Install your local copy into a virtualenv. Assuming you have virtualenv installed,
+1. Install your local copy into a virtualenv. Assuming you have virtualenv installed,
    this is how you set up your fork for local development:
 
         $ cd {{ cookiecutter.project_name }}/
-        $ virtualenv env
+        $ virtualenv --python=python3 env
         $ source env/bin/activate
         $ python setup.py develop
 
-4. Create a branch for local development:
+1. Create a branch for local development and start making your changes locally:
 
         $ git checkout -b name-of-your-bugfix-or-feature
 
-  Now you can make your changes locally.
+1. When you're done making changes, check that your changes pass pylint and the tests:
 
-5. When you're done making changes, check that your changes pass pylint
-   and the tests, including testing other Python versions with tox:
+        $ python setup.py lint
+        $ python setup.py test
+
+{% if cookiecutter.use_tox|lower == "y" -%}
+
+    Use tox for running the validations against different Python versions:
 
         $ tox
 
-  To get tox, just pip install it into your virtualenv.
+    To get tox, just pip install it into your virtualenv.
+{% endif -%}
 
-6. Commit your changes and push your branch to {{ cookiecutter.vs | title }}:
+1. Commit your changes and push your branch to {{ cookiecutter.vs | title }}:
 
         $ git add .
         $ git commit -m "Your detailed description of your changes."
-        $ git push origin name-of-your-bugfix-or-feature
+        $ git push origin name-of-your-bugfix-or-feature-branch
 
-7. Submit a pull request through the {{ cookiecutter.vs | title }} website.
+1. Submit a pull request through the {{ cookiecutter.vs | title }} website.
 
 ## Pull Request Guidelines
 
 Before you submit a pull request, check that it meets these guidelines:
 
 1. The pull request should include tests.
```

Comment:
Do we have somewhere we could link out to for test guidelines?
We kind of have an internal standard, but it isn't really documented anywhere as far as I know...

Reply:
So far we don't have anything

Reply:
I was thinking there might be something external you might know of, sounds like we gotta write our own at some point :)

Reply:
Maybe this one: https://docs.python-guide.org/writing/tests/

Reply:
Yeah, that might be a good starting point. But if that gets added, let's be sure to discourage using unittest and instead prefer pytest (maybe encourage hypothesis use as well since that's useful but a bit more exotic).

Reply:
Added the info to this page